examples/visual-tests/amp-story-player/story-attribution.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Story Attribution</title>
+    <script>
+      // TODO(#17108): Remove this when getMode().test is set.
+      __AMP_TEST = true;
+    </script>
+    <script async src="../../../dist/amp-story-player-v0.js"></script>
+    <link href="../../../build/css/amp-story-player-v0.css" rel='stylesheet' type='text/css'>
```

Comment:
Side note: we really really need to fix these URLs so they work like other components :((